plugin.php

```diff
@@ -149,9 +161,10 @@ function should_prevent_anonymous_access( WP_REST_Server $server, WP_REST_Reques
 	/**
 	 * Prevent access to the root of the REST API.
 	 *
-	 * @param bool $prevent Whether to prevent anonymous access, default false.
+	 * @param bool   $prevent Whether to prevent anonymous access, default false.
```

Comment:
Here, too, I think "allow" instead of "prevent" is better. Someone who returned `true`, thinking "yes, please prevent," would get the opposite, right?

```suggestion
	 * @param bool   $prevent Whether to allow anonymous access to the REST API index. Default false.
```